### DISCUSSION TITLE
Is there a  documentation on how to install for web hook usage or via api?

### DISCUSSION THREAD

**Discussion by scottiesan** [assafelovic/gpt-researcher#986] [Q&A]
Is there a  documentation on how to install for web hook usage or via api?

**Comment by ElishaKay** [comment]
Sup @scottiesan 

Try this one: [Querying the Backend](https://docs.gptr.dev/docs/gpt-researcher/gptr/querying-the-backend)

**Comment by scottiesan** [comment]
Thank you!  That helps!  Seems like I will need to create a bridge in between my N8N and gpt-researcher.   Is there a direct way i can call it from N8N?  Thank!

**Reply by KingSolomun79** [reply]
Have you found a solution for this? I'm trying to make it work with n8n too ,but struggling

**Reply by ntheodorg** [reply]
me too

**Reply by liamgwallace** [reply]
hey i got this running as a mcp server so i can access with n8n. It works ok, but this cant be accessed through workflows, it has to be an ai tool. This works ok, but I have had a deep research time out a few time.

The rough steps i took was to edit the .env and build the Dockerfile in the /mcp-server folder. But i guess you could just run the server.py file.

I have started vibe coding a version of the server.py file that is a fast-api server. That way i can access gpt-researcher with a webhook in a workflow. I am a github noob, but i can try and put a pull request in for my change if you want.
